learning-development/r.qmd

```diff
@@ -592,6 +592,21 @@ Then restart R-Studio and downloads with renv should now succeed.
 
 ---
 
+### Packages not downloading (or not able to connect to other external service from within R, Python, Bash etc.) due to SSL certificate error
+
+---
+
+The DfE firewall currently blocks access to external sites from DfE laptops when 
+connecting from common analytical software such as R or Python. This can prevent
+analysts from accessing important external resources such as package 
+repositories and data API endpoints. If you get an error including text along 
+the lines of "self signed SSL certificate not allowed" or anything similar 
+involving SSL certs., then contact the [Statistics Development Team ](mailto:statistics.development@education.gov.uk) with a copy of the command
```

Comment:
stray whitespace in the team email link text

Reply:
Done